winspool/winspool.go

```diff
@@ -154,7 +154,7 @@ func convertPrinterState(wsStatus uint32) *cdd.PrinterStateSection {
 		}
 		state.VendorState.Item = append(state.VendorState.Item, vs)
 	}
-	if wsStatus&PRINTER_STATUS_OFFLINE != 0 {
+	if wsStatus&PRINTER_STATUS_OFFLINE != 0 || wsAttributes&PRINTER_ATTRIBUTE_WORK_OFFLINE != 0 {
```

Comment:
I ran some tests and compared PRINTER_INFO_2 and PRINTER_INFO_6 of the printer with USB cable connected and disconnected. On Windows 10 and 8, Status and dwStatus are always 0 and the only thing that changes is the value of this attribute. It seems to be set automatically to inform spooler to not despool anything until the printer is connected again.

Can you point me to any sources? https://support.microsoft.com/pl-pl/kb/160456 seems a bit outdated (Windows 95) but it states that when the flag is set, spooler will not despool anything to the physical printer.

When I manually set this attribute on a connected printer using SetPrinter and try to print something it doesn't send data to the printer (as expected). But when i reconnect the USB cable, flag gets automatically reset and printer begins to print.